packages/@aws-cdk/service-spec-types/src/types/database.ts

```diff
@@ -63,7 +63,7 @@ export function emptyDatabase() {
       serviceHasMetric: r.relationship<ServiceHasMetric>('service', 'metric'),
       resourceHasDimensionSet: r.relationship<ResourceHasDimensionSet>('resource', 'dimensionSet'),
       serviceHasDimensionSet: r.relationship<ServiceHasDimensionSet>('service', 'dimensionSet'),
-      hasEvent: r.relationship<HasEvent>('resource', 'event'),
+      resourceHasEvent: r.relationship<ResourceHasEvent>('resource', 'event'),
```

Comment:
You can't rename this, pretty sure this is going to break consumers. Do an alias if you must and deprecate the old name.